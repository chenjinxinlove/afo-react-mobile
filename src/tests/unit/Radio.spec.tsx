import * as React from 'react';
import * as renderer from 'react-test-renderer';
import DemoRadio from '../../pageages/radio/demo';

describe('Button', () => {
  // 测试快照
  it('test snapshots', () => {
    const component = renderer.create(
      <DemoRadio />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
