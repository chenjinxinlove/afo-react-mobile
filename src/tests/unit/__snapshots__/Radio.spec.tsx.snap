// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test snapshots 1`] = `
<div>
  <p>
    基础用法
  </p>
  <div
    className="afo-radio-group"
  >
    <div
      className="afo-radio"
      onClick={[Function]}
    >
      <span
        className="afo-radio__input"
      >
        <input
          className="afo-radio__control"
          disabled={undefined}
          type="radio"
          value="1"
        />
        <i
          className="afo-icon afo-icon-right"
          onClick={undefined}
          style={
            Object {
              "color": undefined,
              "fontSize": "18px",
            }
          }
        />
      </span>
      <span
        className="afo-radio__label afo-radio__label--undefined"
        onClick={[Function]}
      >
        单选框 1
      </span>
    </div>
    <div
      className="afo-radio"
      onClick={[Function]}
    >
      <span
        className="afo-radio__input"
      >
        <input
          className="afo-radio__control"
          disabled={undefined}
          type="radio"
          value="1"
        />
        <i
          className="afo-icon afo-icon-round-border"
          onClick={undefined}
          style={
            Object {
              "color": undefined,
              "fontSize": "18px",
            }
          }
        />
      </span>
      <span
        className="afo-radio__label afo-radio__label--undefined"
        onClick={[Function]}
      >
        单选框 2
      </span>
    </div>
  </div>
  <p>
    禁用状态
  </p>
  <div
    className="afo-radio-group"
  >
    <div
      className="afo-radio afo-radio--disabled"
      onClick={[Function]}
    >
      <span
        className="afo-radio__input"
      >
        <input
          className="afo-radio__control"
          disabled={true}
          type="radio"
          value="1"
        />
        <i
          className="afo-icon afo-icon-right"
          onClick={undefined}
          style={
            Object {
              "color": undefined,
              "fontSize": "18px",
            }
          }
        />
      </span>
      <span
        className="afo-radio__label afo-radio__label--undefined"
        onClick={[Function]}
      >
        单选框 1
      </span>
    </div>
    <div
      className="afo-radio afo-radio--disabled"
      onClick={[Function]}
    >
      <span
        className="afo-radio__input"
      >
        <input
          className="afo-radio__control"
          disabled={true}
          type="radio"
          value="1"
        />
        <i
          className="afo-icon afo-icon-round-border"
          onClick={undefined}
          style={
            Object {
              "color": undefined,
              "fontSize": "18px",
            }
          }
        />
      </span>
      <span
        className="afo-radio__label afo-radio__label--undefined"
        onClick={[Function]}
      >
        单选框 2
      </span>
    </div>
  </div>
</div>
`;
